### ISSUE TITLE
Missing package: wxPython

### ISSUE THREAD

**Issue by bennuttall** [bennuttall/piwheels#54]
It's been reported that the `wxPython` package is missing. It's a case of missing dependencies.

The project's [wiki](https://wiki.wxpython.org/BuildWxPythonOnRaspberryPi) states these as build dependencies:

```
build-essential tk-dev libncurses5-dev libncursesw5-dev libreadline6-dev libdb5.3-dev libgdbm-dev libsqlite3-dev libssl-dev libbz2-dev libexpat1-dev liblzma-dev zlib1g-dev
```

**Comment by bennuttall** [comment]
I've installed these dependencies and kicked off the builders, will see if that's successful.

**Comment by bennuttall** [comment]
That didn't fix it. It looks like something else is missing.

**Comment by bennuttall** [comment]
Now following full set of build instructions... this could take a while.

**Comment by bennuttall** [comment]
> On the piwheels home page, looks like we're building for Python 3.4/3.5. So, wxPython 4.0.3 it is then.

Yes, we have Jessie (3.4) and Stretch (3.5) builders. Try on Stretch and then check the package names match on Stretch, or find alternatives. See our deploy script here: https://github.com/bennuttall/piwheels/blob/master/deploy_slave.sh

**Comment by darrell24015** [comment]
Thanks! I was over-thinking it, for sure.  I'll continue trying to get a successful build and comment back here again.

**Comment by darrell24015** [comment]
Build was successful (second attempt with 32 GB SD card).  Followed all the steps that I mentioned previously on the wiki.wxpython.org site.  I'll look through the deploy script and see if I spot any discrepancies.
